simcon_project/users/views/view_response.py

```diff
@@ -0,0 +1,90 @@
+from django.shortcuts import render, get_object_or_404
+# from django.contrib.auth.decorators import login_required, permission_required
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+
+from users.forms import UpdateFeedback
+from conversation_templates.models import TemplateResponse, TemplateNodeResponse
+
+
+def ViewResponse(request, pk=None):
+    response = get_object_or_404(TemplateResponse, pk=pk)
+
+    nodes = []
+    num_nodes = TemplateNodeResponse.objects.filter(parent_template_response=response).count()
+    for i in range(1, num_nodes+1):
+        if TemplateNodeResponse.objects.get(parent_template_response=response, position_in_sequence=i):
+            nodes.append(TemplateNodeResponse.objects.get(parent_template_response=response,
+                                                          position_in_sequence=i))
+        else:
+            break
+    return render(request, 'view_response.html', {'response_nodes': nodes, 'response': response})
+
+
+# @login_required
```

Comment:
Whats with the commented out decorators?

Reply:
Until the login/permissions thing is fully configured I have them commented, and also for testing
(I wasn't sure if it was fully implemented)

Reply:
devin has it working! i think its safe to talk to him and then uncomment it out, might need to import one thing
@BroughtMyPopcorn